mimic/test/test_maas.py

```diff
@@ -152,24 +152,35 @@ def setUp(self):
         helper = APIMockHelper(self, [MaasApi(["ORD"])])
         self.root = helper.root
         self.uri = helper.uri
-        entity_id = None
-        check_id = None
-        alarm_id = None
-        nt_id = None
-        np_id = None
-        sp_id = None
-        entity_id = self.getXobjectIDfromResponse(self.createEntity('ItsAnEntity'))
-        check_id = self.getXobjectIDfromResponse(self.createCheck('ItsAcheck', entity_id))
-        alarm_id = self.getXobjectIDfromResponse(self.createAlarm('ItsAnAlarm', entity_id, check_id))
-        nt_id = self.getXobjectIDfromResponse(self.createNotification('ItsANotificationTarget'))
-        np_id = self.getXobjectIDfromResponse(self.createNotificationPlan('ItsANotificationPlan'))
-        sp_id = self.getXobjectIDfromResponse(self.createSuppression('ItsASuppression'))
-        self.assertNotEquals(None, entity_id)
-        self.assertNotEquals(None, check_id)
-        self.assertNotEquals(None, alarm_id)
-        self.assertNotEquals(None, nt_id)
-        self.assertNotEquals(None, np_id)
-        self.assertNotEquals(None, sp_id)
+        self.entity_id = self.getXobjectIDfromResponse(self.createEntity('ItsAnEntity'))
+        self.check_id = self.getXobjectIDfromResponse(self.createCheck('ItsAcheck',
+                                                      self.entity_id))
+        self.alarm_id = self.getXobjectIDfromResponse(self.createAlarm('ItsAnAlarm',
+                                                      self.entity_id,
+                                                      self.check_id))
+        self.nt_id = self.getXobjectIDfromResponse(self.createNotification('ItsANotificationTarget'))
+        self.np_id = self.getXobjectIDfromResponse(self.createNotificationPlan('ItsANotificationPlan'))
+        self.sp_id = self.getXobjectIDfromResponse(self.createSuppression('ItsASuppression'))
+
+    def test_resource_ids(self):
+        """
+        MAAS sets IDs for reources it created by prefixing it with
+        the first two characters of the resource type.
+        For example an entity id is prefixed with 'en'.
```

Comment:
This whole paragraph looks like it's wrapped to a way-too-narrow margin, too.  Maybe tell sublime to re-wrap it to 79?